src/plugins/emergencyanalysis_event.conf

```diff
@@ -2,5 +2,6 @@ EVENT=report_EmergencyAnalysis
     rpm -q abrt > abrt_installed_version 2>/dev/null
     rpm -q libreport > libreport_version 2>/dev/null
     rpm -q satyr > satyr_version 2>/dev/null
-    EXCLUDE_FROM_REPORT="" reporter-upload
+    echo -e "An archive will be created.\nThis archive is accessible only by ABRT developers.\nTherefore if you try to open the produced web location you will not be able to see it."
```

Comment:
>An archive will be created

IMO, that implies that the archive will be available locally, so maybe it would be better to just say something like “Uploading report data. It will not be accessible outside of the ABRT development team.”?

Also, is there actually a point in printing the URL not at debug level? Adding a secondary debug print in https://github.com/abrt/libreport/blob/779b16202c3fe990f6ecda17085c436899340e12/src/lib/curl.c#L719 for the URL makes more sense to me. And why is that a warning?